pipeline.py

```diff
@@ -432,6 +453,20 @@ def ilab_pipeline(
     model_pvc_delete_task = DeletePVC(pvc_name=model_pvc_task.output)
     model_pvc_delete_task.after(final_eval_task)
 
+    generate_metrics_report_task = generate_metrics_report_op()
+    generate_metrics_report_task.after(output_mt_bench_task, final_eval_task)
+    generate_metrics_report_task.set_caching_options(False)
+    mount_pvc(
+        task=generate_metrics_report_task,
+        pvc_name=output_pvc_task.output,
+        mount_path="/output",
+    )
+
+    output_pvc_delete_task = DeletePVC(pvc_name=output_pvc_task.output)
+    output_pvc_delete_task.after(
+        output_model_task, output_mt_bench_task, final_eval_task
```

Comment:
Can you please add the dependency on the new artifact upload tasks?

`output_mmlu_branch_task`, `output_mt_bench_branch_task`

I think it should also run only after on `generate_metrics_report_task` finishes, so maybe that's also a dependency...